packages/moonstream-components/src/core/providers/AnalyticsProvider/index.js

```diff
@@ -108,10 +108,10 @@ const AnalyticsProvider = ({ children, mixpanelToken }) => {
     }
     const urlForUnmount = router.nextRouter.pathname;
     const closeListener = () => {
-      mixpanel?.track(MIXPANEL_EVENTS.PAGEVIEW_DURATION, {
-        url: urlForUnmount,
-        isBeforeUnload: true,
-      });
+      // mixpanel?.track(MIXPANEL_EVENTS.PAGEVIEW_DURATION, {
```

Comment:
Uncomment for now - fix in separate PR but let's not change analytics behavior like this in this release.